src/export/rawdk.ml

```diff
@@ -25,7 +25,7 @@ let rec term : p_term pp = fun ppf t ->
   | P_Expl u -> out ppf "(%a)" term u
   | P_Type -> out ppf "Type"
   | P_Wild -> out ppf "_"
-  | P_NLit i -> string ppf i
+  | P_NLit (path, i) -> out ppf "%a%s" Dk.path path i
```

Comment:
Integer literals are undefined in Dedukti. Actually we should fail (already in master) like for string literals.